lib/ecs.go

```diff
@@ -155,11 +166,118 @@ func GetPendingTasksCount(awsSess *session.Session, cluster string) int64 {
 		os.Exit(1)
 	}
 
-	var pendingTasks int64
+	return descResult.Services
+}
 
-	for _, service := range descResult.Services {
-		pendingTasks += *service.PendingCount
+func GetMemoryCpuNeededForEcsServices(awsSess *session.Session, ecsServices []*ecs.Service) (int64, int64) {
+	var memoryNeeded int64 = 0
+	var cpuNeeded int64 = 0
+	var largestServiceMemory int64 = 0
+	var largestServiceCpu int64 = 0
+
+	svc := ecs.New(awsSess)
+
+	for _, service := range ecsServices {
+		if *service.DesiredCount > 0 {
```

Comment:
How about avoiding the extra indentations with ...
```
  if *service.DesiredCount == 0 {
    continue
  }

  taskDef, err := ...
```

Reply:
fixed